src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/TransformChain.cs

```diff
@@ -52,23 +52,23 @@ public int Count
             {
                 if (index >= _transforms.Count)
                     throw new ArgumentException(SR.ArgumentOutOfRange_IndexMustBeLess, nameof(index));
-                return (Transform)_transforms[index];
+                return (Transform)_transforms[index]!;
             }
         }
 
         // The goal behind this method is to pump the input stream through the transforms and get back something that
         // can be hashed
-        internal Stream TransformToOctetStream(object inputObject, Type inputType, XmlResolver resolver, string baseUri)
+        internal Stream? TransformToOctetStream(object? inputObject, Type inputType, XmlResolver? resolver, string? baseUri)
```

Comment:
When will this method ever return null? 
In two cases it just returns `new MemoryStream`, 
and in one it returns `if (currentInput is Stream) return currentInput as Stream;` (which is also never null)

Also upstream calls of this method (in `EncryptedXml.cs`) don't expect it to return null

Reply:
```C#
if (currentInput is Stream inputStream)
{
    return inputStream;
}
```

Reply:
Of course! It's been a long day...